test/wrb.js

```diff
@@ -299,7 +297,7 @@ contract("WitnetRequestBoard", ([
           requestId, 0, resultHex,
           { from: owner, gasPrice: 1 }
         ),
-        "Data request transaction cannot be zero"
+        "tx hash cannot be zero"
```

Comment:
The old message was better because `tx hash` can also mean ethereum transaction hash, or tally transaction hash. If the string length is a problem, try "DR tx hash cannot be zero".

Reply:
Right, it'll change it to: "DR tally tx hash cannot be zero"